### PR TITLE
release(falco): upgrade to Falco 0.24.0 and bump chart version to 1.2.0

### LINKED ISSUE #48: Update the Falco's Helm chart once Falco 0.24.0 has been released
**Motivation**

We just need to keep the chart in sync.

**Feature**

- [x] Merge https://github.com/falcosecurity/charts/pull/29 (Unix socket support)
- [x] Update the ruleset (grab files from the Falco's main repository once the release has a git tag)
- [x] Update `falco/Chart.yaml` with `appVersion: 0.24.0`
- [x] Release a new Falco chart version (bump to 1.2.0 since a new feature has been added)

**Alternatives**

No alternatives.

**Additional context**

We just have to keep this on hold until Falco 0.24.0 has been released (scheduled for tomorrow).
/assign

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

/kind feature

/kind chart-release

**Any specific area of the project related to this PR?**

/area falco-chart

**What this PR does / why we need it**:

This PR follows up the recent Falco 0.24.0 release. It also: 
- updates the ruleset, grabbing files from the [git tag 0.24.0 of the falcosecurity/falco repository](https://github.com/falcosecurity/falco/releases/tag/0.24.0).
- set default threadiness to 0 ("auto" behavior) for the gRPC server

/cc @nibalizer 

**Which issue(s) this PR fixes**:

Fixes #48 

**Special notes for your reviewer**:

This is the first PR using the customized Pull request template. Moreover, it is also the first PR with [this setting](https://github.com/falcosecurity/test-infra/pull/150) to allow @poiana pushing into the `gh-pages` branch.
/cc @leodido 

**Checklist**

- [x] Chart Version bumped
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by leogr** [comment]
I have added the new default gRPC threadiness setting (`0`).

Ready to be merged.

/hold cancel

File: falco/README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The following table lists the configurable parameters of the Falco chart and the
...
 | `falco.grpc.enabled`                            | Enable the Falco gRPC server                                                                                       | `false`                                                                                                                                   |
-| `falco.grpc.threadiness`                        | Number of threads (and context) the gRPC server will use                                                           | `8`                                                                                                                                       |
+| `falco.grpc.threadiness`                        | Number of threads (and context) the gRPC server will use, `0` by default, which means "auto"                                                           | `0`                                                                                                                                       |
```
Comment (leodido) [inline falco/README.md:117]: A leftover tab?

**Comment by nibalizer** [comment]
Go release go!

File: falco/README.md (hunk 1 above)
Comment (leogr) [reply falco/README.md:117]: Thanks for noticing this, I will fix it next time :)

**Pull request merged**